<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlidingBarChart</class>
 <widget class="QWidget" name="SlidingBarChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2090</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sliding Bar Chart</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="m_bar_charts_container_widget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>510</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1761</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="m_bar_charts_container_widget_horizontal_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_y_axis_percentile_data_labels_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="m_y_axis_100_percentile_data_label">
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_y_axis_80_percentile_data_label">
      <property name="text">
       <string>8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_y_axis_60_percentile_data_label">
      <property name="text">
       <string>6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_y_axis_40_percentile_data_label">
      <property name="text">
       <string>4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_y_axis_20_percentile_data_label">
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_y_axis_0_percentile_data_label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="xAxisSeperator">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>18</y>
     <width>1</width>
     <height>273</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yAxisSeperator">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>480</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblgraphSeperator_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>38</y>
     <width>481</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblgraphSeperator_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>88</y>
     <width>481</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblgraphSeperator_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>138</y>
     <width>481</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblgraphSeperator_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>188</y>
     <width>481</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblgraphSeperator_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>238</y>
     <width>481</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblFaderLeft">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>60</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblFaderRight">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>100</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblgraphSeperator_10</zorder>
  <zorder>lblgraphSeperator_2</zorder>
  <zorder>lblgraphSeperator_6</zorder>
  <zorder>lblgraphSeperator_4</zorder>
  <zorder>lblgraphSeperator_8</zorder>
  <zorder>m_bar_charts_container_widget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>lblFaderRight</zorder>
  <zorder>lblFaderLeft</zorder>
  <zorder>xAxisSeperator</zorder>
  <zorder>yAxisSeperator</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
